classes/actions/ValidationOrderActions.php

```diff
@@ -626,15 +627,15 @@ public function addTentative()
                         'OrderPayment is not created due to a PrestaShop, please verify order state configuration is loggable (Consider the associated order as validated). We try to create one with charge id ' .$this->conveyor['chargeId'] . ' on payment.',
                         'Order',
                         $orderId,
-                        'validation'
+                        'ValidationOrderActions - addTentative'
                     );
                     $order = new Order($orderId);
                     if (!$order->addOrderPayment($this->conveyor['amount'], null, $this->conveyor['chargeId'])) {
                         ProcessLoggerHandler::logError(
                             'PaymentModule::validateOrder - Cannot save Order Payment',
                             'Order',
                             $orderId,
-                            'validation'
+                            'ValidationOrderActions - addTentative'
```

Comment:
At the begining of a process, for instance of a hook or front controller you can call
`ProcessLoggerHandler::openLogger('webhook:' . $event->type)`
After that, all ProcessLoggerHandler::logInfo (or logSuccess or logError) doesn't need the forth paramter and use the previous process name defined in a registry in openLogger method.